amivapi/settings.py

```diff
@@ -35,6 +35,7 @@
 X_DOMAINS = '*'
 X_HEADERS = ['Authorization', 'If-Match', 'Content-Type']
```

Comment:
I think it makes sense to add `Cache-Control` here to allow users to set their own cache settings.

Reply:
Why should this be allowed? 
I don't see why the client should specify this header in a request.

Reply:
I agree with @hermannsblum -- as far as I understand, the API will send out a cache-control header to the client, but if the client were to respond using this (provided) header, the browser would actually prevent the client from making the request (since `Cache-Control` is not one of the allowed cross domain headers)